src/react/components/common/tagsInput/tagsInput.tsx

```diff
@@ -156,6 +156,12 @@ export default class TagsInput extends React.Component<ITagsInputProps, ITagsInp
         tag.text = this.ReactTagHtml(tag.id, tag.color);
     }
 
+    private getReactTags(props): IReactTag[] {
+        const tags = (props.tags) ? props.tags : props.value;
```

Comment:
Interesting - so I'm assuming in the context of the form the tags are passed by the `value` prop, but in the editor page tags are passing in the `tags` props.  This introduces some complexity.  Let's chat.

Reply:
Exactly. This was part of the reason the tests were failing when trying to mount the project form.

Reply:
And I don't know for sure if it's the editor page that passes it in this way. This was mainly an artifact of testing. Maybe if I remove it from `Provider` and `Router` this won't happen?

Reply:
I figured this out. Because of the custom widget, `props.value` was being used as the default for passing the tags data to the component. By augmenting the existing `createUiSchema`, I was able to actually instantiate the `TagsInput` component and specify the `onChange` function and the `tags` data